charts/dynamoai-api/values.yaml

```diff
@@ -2,6 +2,20 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 global:
+  azure:
+    images:
+      apiInitCheckers:
+        registry: "abc"
+        image: "def"
+        tag: "123"
+      api:
+        registry: "abc"
+        image: "def"
+        tag: "123"
+      dbMigrations:
+        registry: "abc"
+        image: "def"
+        tag: "123"
```

Comment:
would we merge these changes?

```
"{{ if .Values.global.azure.images.api.registry }}{{ .Values.global.azure.images.api.registry }}/{{ .Values.global.azure.images.api.image }}:{{ .Values.global.azure.images.api.tag }}{{ else }}{{ .Values.api.image.registry }}/{{ .Values.api.image.repository }}:{{ .Values.api.image.tag }}{{ end }}"
```

provided the above code, it would find the image and registry and would try to fetch it from wrong place, isn't it?

I believe we might help it override with some other file, but do we need this here?